### PR TITLE
feat: updating reactor formula request params docs

### PR DESCRIPTION
**Pull request by jleon15**
## Description

- Update reactor formula request params section

## Testing required outside of automated testing?

- [x] Not Applicable

### Screenshots (if appropriate):
image

- [] Not Applicable

## Rollback / Rollforward Procedure

- [x] Roll Forward
- [ ] Roll Back

## Reviewer Checklist

- [ ] Description of Change
- [ ] Description of outside testing if applicable.
- [ ] Description of Roll Forward / Backward Procedure
- [ ] Documentation updated for Change

### CODE REVIEW

File: source/includes/api-reference/_reactor_formulas.md [hunk 1]
```diff
@@ -57,6 +60,8 @@ a Reactor Formula should define the following request parameters:
 | `card.expiration_year`  | *number* | false    |
 | `card.cvc`              | *string* | true     |
 
+As you can see, only the primitive typed request parameters are defined. You could even omit them completely and they would still be forwarded to the Reactor, but no validation would be applied.
```
Comment (koalaty-code) [inline source/includes/api-reference/_reactor_formulas.md:63]: What happens to type conversion if request parameters are undefined? Is everything a JSON stringified?

Comment (bweber) [reply source/includes/api-reference/_reactor_formulas.md:63]: We look at the original data type and attempt to convert to that. If that fails, we fall back to a string value.

Comment (armsteadj1) [inline source/includes/api-reference/_reactor_formulas.md:63]:
```suggestion
As you can see, only the primitive typed request parameters are defined. You could even omit them completely and they would still be forwarded to the Reactor and no validation would be applied.
```

File: source/includes/api-reference/_reactor_formulas.md [hunk 2]
```diff
@@ -46,8 +46,11 @@ Elements of a Reactor Formula's `request_parameters` array have the following sc
 | `optional`    | false    | *boolean* | `false` | If the request parameter is optional when executing the reactor                        |
 
 Request parameters are intended to define any parameters that will be provided to a Reactor at request-time, and may change across Reactor invocations.
-Complex objects can be passed within the `args` property to a Reactor, and these complex request parameters can be defined by dot-separating levels of the object hierarchy.
-For example, to pass a `card` object to a Reactor whose schema matches the [Card Object](#atomic-cards-atomic-card-object-card-object) stored within an [Atomic Card](#atomic-cards-atomic-card-object) token, 
+Complex objects properties can be passed within the `args` property to a Reactor, and they can be defined by dot-separating levels of the object hierarchy.
+
+Any `args` property that is not associated with a request parameter is still forwarded to the reactor. This allows you to provide complete complex objects including arrays, in which no type checking is applied. For instance, if no request parameters are declared, it means you can provide any payload when invoking the reactor.
```
Comment (armsteadj1) [inline source/includes/api-reference/_reactor_formulas.md:51]:
```suggestion
Any `args` property not associated with a request parameter is still forwarded to the reactor. This allows you to provide complete complex objects including arrays, in which no type checking is applied. For instance, if no request parameters are declared, it means you can provide any payload when invoking the reactor. 
```

**Review by armsteadj1 — CHANGES_REQUESTED** [review]

**Review by armsteadj1 — APPROVED** [review]

**Pull request merged**